From 58dcb1ebb77a50ed0220947d0d67d246383a9204 Mon Sep 17 00:00:00 2001
Message-ID: <58dcb1ebb77a50ed0220947d0d67d246383a9204.1711239854.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1711239854.git.sharlatanus@gmail.com>
References: <cover.1711239854.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Fri, 22 Mar 2024 21:26:16 +0000
Subject: [PATCH 05/12] gnu: clipman: Fix source and home-page URL.

The URI https://github.com/yory8/clipman/ no longer exists. The new
project's URL was obtained from the Nix package.

* gnu/packages/xdisorg.scm (clipman) [source]: Fix URL.
[home-page]: As above.

Change-Id: I20e9f22780586ce14d925d5cf106e70de8209e1e
---
 gnu/packages/xdisorg.scm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/xdisorg.scm b/gnu/packages/xdisorg.scm
index 46c9305e65..ba7c5b942e 100644
--- a/gnu/packages/xdisorg.scm
+++ b/gnu/packages/xdisorg.scm
@@ -3227,7 +3227,7 @@ (define-public clipman
     (source (origin
               (method git-fetch)
               (uri (git-reference
-                    (url (string-append "https://github.com/yory8/" name "/"))
+                    (url "https://github.com/chmouel/clipman")
                     (commit (string-append "v" version))))
               (file-name (git-file-name name version))
               (sha256 (base32
@@ -3260,7 +3260,7 @@ (define-public clipman
 clipman store} (or @command{exec wl-paste -t text --watch clipman store 1>>
 PATH/TO/LOGFILE 2>&1 &} to log errors) at the beginning of wherever you
 initialize programs.")
-    (home-page "https://github.com/yory8/clipman")
+    (home-page "https://github.com/chmouel/clipman")
     (license license:gpl3)))
 
 (define-public kbdd
-- 
2.41.0

